configure.ac

```diff
@@ -1077,7 +1077,7 @@ AC_CACHE_SAVE
 # visible again
 #
 ###########################################################
-OPAL_SETUP_FT_OPTIONS
+#OPAL_SETUP_FT_OPTIONS
```

Comment:
This will not do what you want.  You want dnl (to prevent the m4 macro from expanding) rather than #, which will just make the macros commented out.

But the bigger question is why not just remove the line?